applications/addons/controllers/class.addoncontroller.php

```diff
@@ -665,7 +670,7 @@ public function browse($FilterToType = '', $Sort = '', $VanillaVersion = '', $Pa
             $Checked = $Session->getPreference('Addons.FilterChecked');
         }
 
-        if (!array_key_exists($FilterToType, AddonModel::$TypesPlural)) {
+        if (!array_key_exists($FilterToType, AddonModel::$TypesPlural) && $FilterToType != 'plugins,applications') {
```

Comment:
This feels icky. What about something like

    !array_key_exists($FilterToType, AddonModel::$TypesPlural + ['plugins,applications' => true])

I would assume moving this to a model method would be cleanest. Baby steps, I know.